packages/node_modules/@webex/plugin-meetings/src/meeting-info/util.js

```diff
@@ -130,9 +135,17 @@ MeetingInfoUtil.generateOptions = (from) => {
     options.type = _SIP_URI_;
     options.destination = destination;
   }
-  else {
+  else if (MeetingInfoUtil.isConversationUrl(destination, webex)) {
     options.type = _CONVERSATION_URL_;
-    options.destination = `${conversationServiceUrl}/${CONVERSATIONS}/${destination}`;
+    if (destination.includes(`${CONVERSATION_URL_API}${CONVERSATION_URL_GET_API}`)) {
+      options.destination = destination;
+    }
+    else {
+      options.destination = destination.replace(CONVERSATION_URL_API, `${CONVERSATION_URL_API}${CONVERSATION_URL_GET_API}`);
+    }
```

Comment:
I don't understand this logic. if destination doesn't include `conversation/api/v1/conversations` replace `conversation/api/v1` with `conversation/api/v1/conversations`. Is this specifically for the case where destination was an id? If so, I think I would rather check if destination is a UUID and then do the logic, or make it a breaking change. this appears to be making it backwards compatible (which is ok for long standing APIs, but for new stuff, I don't mind s hipping breaking changes)

Reply:
it's to allow users to pass a conversation id with and without the GET API included. I'm fine with doing it with just a conversation_url.

Reply:
Correct. So here we are making it backward compatible, but other places we are removing support for it. Like this change

```
 // If the destination is a UUID, assume it is an internal conversation ID.
  if (UUID_REG.test(dest)) {
    return dest;
  }

  // Next, assume it is a valid Hydra room ID.
  // If decoding fails, it is a SIP or other destination.
  // if the id is not present we will use the default destination
  return deconstructHydraId(dest).id || dest;
  return dest;
```

I'm fine just enforcing convoUrl and no longer accepting convoId as a param. @adamweeks? Otherwise we do need to default to the cluster url. I would prefer that destination check explicitly for a UUID instead of checking if it contains the url if that's the case we are checking for.

Reply:
If the destination doesn't have `conversations` in it. What does it look like? It's not a URL to a conversation.

Reply:
I think i can dig in, we might not even need to splice in the /conversations portion, i will check

Reply:
I agree, we should not support UUID convo id at all.